example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml

```diff
@@ -4486,6 +4486,128 @@ spec:
                 baseImage:
                   description: Thanos base image if other than default.
                   type: string
+                grpcServerTlsConfig:
+                  description: 'GRPCServerTLSConfig configures the GRPC server from
+                    which Thanos Querier reads recorded rule data. Note: currently
+                    only the CAFile, CertFile, and KeyFile fields are supported. Maps
+                    to the ''--grpc-server-tls-*'' CLI args.'
+                  properties:
+                    ca:
+                      description: Stuct containing the CA cert to use for the targets.
+                      properties:
+                        configMap:
+                          description: ConfigMap containing data to use for the targets.
+                          properties:
+                            key:
+                              description: The key to select.
+                              type: string
+                            name:
+                              description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                TODO: Add other useful fields. apiVersion, kind, uid?'
+                              type: string
+                            optional:
+                              description: Specify whether the ConfigMap or its key
```

Comment:
missing trailing period

Reply:
This one is coming from the K8S libraries so would have to be fixed there.